test/zzz_od/game_settings/game_settings_profile_service/test_validate_profile.py

```
@@ -0,0 +1,67 @@
+        encoding="utf-8-sig",
+    )
+
+    with pytest.raises(GameSettingsProfileError, match="只允许"):
+        _service().validate_profile(str(profile_path))
+
+
+def test_validate_profile_rejects_non_reg_file(tmp_path: Path) -> None:
+    profile_path = tmp_path / "profile.txt"
+    profile_path.write_text("not a registry profile", encoding="utf-8")
+
+    with pytest.raises(GameSettingsProfileError, match=".reg"):
```

Ruff (0.16.1)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)